### PR TITLE
HZC-2717: Add the ability to download custom classes

### PR DESCRIPTION
**Pull request by nosan**
Screenshot 2022-05-16 at 12 27 31

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -17,15 +17,14 @@ require (
 	github.com/golang-jwt/jwt/v4 v4.1.0 // indirect
 	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
 	github.com/google/uuid v1.3.0
-	github.com/hazelcast/hazelcast-cloud-sdk-go v1.3.1
+	github.com/hazelcast/hazelcast-cloud-sdk-go v1.3.2-0.20220510130800-fdf9da1d0708
```
Comment (sdreger) [inline go.mod:20]: Don't forget to update the version to `v1.3.2`

**Review by sdreger — APPROVED** [review]

File: cmd/serverless_cluster.go [hunk 2]
```diff
@@ -34,23 +38,26 @@ func newServerlessClusterCreateCmd() *cobra.Command {
 				createClusterInputParams.ClusterType = models.Serverless
 			}
 			client := internal.NewClient()
-			cluster := internal.Validate(client.ServerlessCluster.Create(context.Background(), &createClusterInputParams)).(*models.Cluster)
-			color.Green("Cluster %s is creating. You can check the status using hzcloud serverless-cluster list.", cluster.Id)
+			cluster := internal.Validate(client.ServerlessCluster.Create(context.Background(),
+				&createClusterInputParams)).(*models.Cluster)
+			color.Green("Cluster %s is creating. You can check the status using hzcloud serverless-cluster list.",
```
Comment (huseyinbabal) [inline cmd/serverless_cluster.go:43]: What about
```
bold := color.New(color.Bold)
color.Green("Cluster %s is being created. You can check the status using %s.", bold.Sprintf("hzcloud serverless-cluster list"), cluster.Id)
```

**Review by huseyinbabal — APPROVED** [review]
I am ok with minor comments

File: cmd/serverless_cluster.go (hunk 2 above)
Comment (nosan) [reply cmd/serverless_cluster.go:43]: it is not my codebase. just formatting issue :(

**Pull request merged**